bundles/core/src/main/java/com/adobe/cq/wcm/core/components/services/link/PathProcessor.java

```diff
@@ -64,4 +67,13 @@
      * @return the external link of the given path
      */
     @NotNull String externalize(@NotNull String path, @NotNull SlingHttpServletRequest request);
+
+
+    /**
+     * Process the HTML attributes for the {@link com.adobe.cq.wcm.core.components.internal.link.LinkHandler}
+     * @param path the path of the linked resource
+     * @param htmlAttributes the origin HTML attributes of the link
```

Comment:
I'd rephrase it as: the HTML attributes to be processed as they can be any attributes, not just the original ones.